src/scripts/TalismanStoreDevtools.js

```diff
@@ -28,4 +29,62 @@ export default class TalismanStoreDevtools extends StoreDevtools {
 		};
 		return await browser.runtime.sendMessage(request);
 	}
+
+	async getAllSitemaps(projectId) {
+		const request = {
+			getAllSitemaps: true,
+			projectId: projectId,
+		};
+		return this._getAllSitemapsResponseHandler(await browser.runtime.sendMessage(request));
+	}
+
+	async getAllProjects() {
+		const request = {
+			getAllProjects: true,
+		};
+		return await browser.runtime
+			.sendMessage(request)
+			.then(res => res.data.data.paginationProject.listProject);
```

Comment:
I'd move this `.data.data.paginationProject.listProject` to StoreTalismanApi, where the query is